WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalPostProviderHelper.kt

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.localcontentmigration
+
+import org.wordpress.android.fluxc.store.PostStore
+import org.wordpress.android.fluxc.store.SiteStore
+import org.wordpress.android.localcontentmigration.LocalContentEntityData.PostData
+import org.wordpress.android.localcontentmigration.LocalContentEntityData.PostsData
+import javax.inject.Inject
+
+class LocalPostProviderHelper @Inject constructor(
+        private val siteStore: SiteStore,
+        private val postStore: PostStore,
+    ): LocalDataProviderHelper {
+    override fun getData(localSiteId: Int?, localEntityId: Int?): LocalContentEntityData {
+        localEntityId?.let { localPostId ->
+            val post = postStore.getPostByLocalPostId(localPostId)
+            return PostData(post = post)
+        } ?: run {
+            val site = siteStore.getSiteByLocalId(localSiteId!!)
```

Comment:
Same question about the `!!`

Reply:
Same as: https://github.com/wordpress-mobile/WordPress-Android/pull/17398#discussion_r1011026351